test/commands/force/org/open.test.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { $$, expect, test } from '@salesforce/command/lib/test';
+import { Org, MyDomainResolver, Messages } from '@salesforce/core';
+import { stubMethod } from '@salesforce/ts-sinon';
+import * as utils from '../../../../src/shared/utils';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-org', 'open');
+
+const orgId = '000000000000000';
+const username = 'test@test.org';
+const testPath = '/lightning/whatever';
+const testInstance = 'https://cs1.my.salesforce.com';
+const accessToken = 'testAccessToken';
+
+const expectedUrl = `${testInstance}/secur/frontdoor.jsp?sid=${accessToken}&retURL=${encodeURIComponent(testPath)}`;
+
+const testJsonStructure = (response: object) => {
+  expect(response).to.have.property('url');
+  expect(response).to.have.property('username').equal(username);
+  expect(response).to.have.property('orgId').equal(orgId);
+  return true;
+};
+
+describe('open commands', () => {
+  const spies = new Map();
+  afterEach(() => spies.clear());
+
+  beforeEach(async function () {
+    $$.SANDBOX.restore();
```

Comment:
Why not restore in `afterEach`?

Reply:
🤷🏻 sometimes there's an `afterEach`, but there's almost always a beforeEach.  So it's mostly consistency (same place each time so it gets done).

Reply:
Consistency is a wonderful thing, but IMHO, teardown tasks belong in teardown methods.